modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/PartitionAccess.java

```diff
@@ -135,6 +135,27 @@ public interface PartitionAccess {
      */
     long maxLastAppliedTerm();
 
+    /**
+     * Returns the start time of the known lease for this replication group.
+     *
+     * @return Lease start time.
+     */
```

Comment:
```suggestion
    /** Returns the start time of the known lease for this replication group. */
```

Reply:
One-line javadocs are not used in this interface, so the change would not align with the existing code